### ISSUE TITLE
Does not work for multi-inputs

### ISSUE THREAD

**Issue by philipperemy** [philipperemy/keras-visualize-activations#1]
```
    m = get_multi_inputs_model()
    m.compile(optimizer='adam',
              loss='binary_crossentropy')

    inp_a = np.random.uniform(size=(100, 10))
    inp_b = np.random.uniform(size=(100, 10))
    inp_o = np.random.randint(low=0, high=2, size=(100, 1))
    m.fit([inp_a, inp_b], inp_o)

    from read_activations import *

    get_activations(m, [inp_a[0:1], inp_b[0:1]], print_shape_only=True)
```

**Comment by philipperemy** [comment]
Currently working on it

**Comment by philipperemy** [comment]
Fixed with 70cf77c1a32b0a06b8c8c0e5827293c06219ecff
